nat-lab/docker-compose.yml

```diff
@@ -158,6 +182,7 @@ services:
       CLIENT_GATEWAY_PRIMARY: 192.168.101.254
     extra_hosts:
       - "photo-album:10.0.80.80"
+      - "photo-album:2001:0db8:85a4::adda:edde:0005"
```

Comment:
is this correct? IPv6 address of a photo album as `extra_hosts` should be added to all containers which has IPv6 connectivity (e.g. `nat-lab-open-internet-client-dual-stack-1`) the con-client-01 -> not sure if this is relevant.

Reply:
Good catch, fixed!